exercises/retree/example.py

```diff
@@ -0,0 +1,17 @@
+def treeFromTraversals(preorder, inorder):
```

Comment:
Forgive me; I've been bouncing between too many different languages lately. My previous suggestion was incorrect, this should be `snake_case`.

```suggestion
def tree_from_traversals(preorder, inorder):
```